.github/workflows/build_ubuntu.yml

```diff
@@ -60,10 +55,7 @@ jobs:
       - name: Install mandatory system libraries
         run: |
           sudo apt-get update --fix-missing
-          sudo apt-get install libjsoncpp-dev libgtest-dev libboost-mpi-dev libboost-program-options-dev libtbb-dev
-          cd /usr/src/googletest/
-          sudo cmake .
-          sudo cmake --build .
+          sudo apt-get install -y libjsoncpp-dev libtbb-dev
```

Comment:
did move the install of `libgtest-dev libboost-mpi-dev libboost-program-options-dev` elsewhere?

Reply:
Since we build boost from source it is not necessary to install it. It prevents possible conflicts